packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/upgrade/steps/place-order/confirm/confirm.controller.js

```diff
@@ -7,22 +7,50 @@ export default class {
     itemName,
     itemType,
     prices,
+    isDowngrade,
   ) {
     this.$translate = $translate;
     this.$uibModalInstance = $uibModalInstance;
     this.hasDefaultMeansOfPayment = hasDefaultMeansOfPayment;
     this.itemName = itemName;
     this.itemType = itemType;
     this.prices = prices;
+    this.isDowngrade = isDowngrade;
   }
 
-  $onInit() {
-    this.addingOrDeducting =
-      (this.prices.hourly.exists && this.prices.hourly.value > 0) ||
-      (this.prices.monthly.exists && this.prices.monthly.value > 0)
-        ? 'adding'
-        : 'deducting';
+  getTitle() {
+    if (this.isDowngrade ) {
```

Comment:
```suggestion
    if ( this.isDowngrade ) {
```
Spacing look weird, did prettier/eslint ran on those files ?